ibis/backends/tests/test_client.py

```diff
@@ -202,3 +203,263 @@ def test_separate_database(con, alternate_current_database, current_data_db):
     db = con.database(current_data_db)
     assert db.name == current_data_db
     assert tmp_db.name == alternate_current_database
+
+
+def create_temp_table_with_schema(con, temp_table_name, schema):
```

Comment:
`_create_temp_table_with_schema`  (starting with underscore) to better show that this is not a test, not something imported from anywhere.

Reply:
Done!